style.css

```diff
@@ -23,17 +42,167 @@ ul {
   list-style: none;
 }
 
-a {
-  text-decoration: none;
-  color: white;
-}
-
-a:hover {
-  color: #333;
-}
-
 /*
     ==============================================
                 ****** GLOBAL ******
     ==============================================
 */
+
+/* footer */
+
+@import url("https://cdn.jsdelivr.net/npm/bootstrap-icons@1.4.0/font/bootstrap-icons.css");
+@import url("https://fonts.googleapis.com/css2?family=Roboto+Condensed:wght@300;400;500;700&display=swap");
+
+h1 {
+  font-family: "Roboto Condensed", sans-serif;
+}
+
+html,
+body {
+  height: 100%;
+}
+
+.main {
+  min-height: 50%;
+}
+
+.sub {
+  border-radius: 10px;
+}
+
+::before {
+  box-sizing: border-box;
+}
+
+.first-bottom {
+  background-color: #f72210;
+  border-bottom: 2px solid #eee;
+  width: 66px !important;
+  position: absolute;
+  top: 88px;
+  right: 0;
+  content: "";
+  display: block;
+  border-left: 12px solid white;
+  border-top: 13px solid #f72210;
+}
+
+.content {
+  position: fixed;
+  bottom: 0;
+}
```

Comment:
- [ ] You can make your footer stick to the bottom by adding the following bootstrap class `fixed-bottom`